web/src/main/java/com/microsoft/applicationinsights/web/internal/WebRequestTrackingFilter.java

```diff
@@ -250,30 +272,28 @@ private void setKeyOnTLS(String key) {
     public WebRequestTrackingFilter() {
     }
 
-    private synchronized void initialize(FilterConfig filterConfig) {
-
+    private synchronized void initializeAgentIfAvailable(FilterConfig filterConfig) {
         StopWatch sw = StopWatch.createStarted();
 
         //If Agent Jar is not present in the class path skip the process
-        if (!CommonUtils.isClassPresentOnClassPath(AGENT_LOCATOR_INTERFACE_NAME,
-            this.getClass().getClassLoader())) {
+        if (!CommonUtils.isClassPresentOnClassPath(AGENT_LOCATOR_INTERFACE_NAME, this.getClass().getClassLoader())) {
```

Comment:
Why this style change is part of this PR?

Reply:
Just to see if you'd notice :P

Reply:
I didn't but appreciate that you changed it back! These were nit picks but I learned from you in my last PRs. Nits go unnoticed and slowly start creeping in :)